#include <exception>
using std::exception;
#include "LogicExceptions.h"

